sky/provision/kubernetes/utils.py

```diff
@@ -812,6 +812,10 @@ def setup_ssh_jump_svc(ssh_jump_name: str, namespace: str,
     # Fill in template - ssh_key_secret and ssh_jump_image are not required for
     # the service spec, so we pass in empty strs.
     content = fill_ssh_jump_template('', '', ssh_jump_name, service_type.value)
+
+    # Add custom metadata from config
+    merge_custom_metadata(content['service_spec']['metadata'])
```

Comment:
Q: If we have multiple users sharing the same jump pod, and they have different metadata field setup in their `~/.sky/config.yaml`. What will happen?

Reply:
Great question - the metadata specified during the first creation of the ssh jump pod will be used throughout it's life. Other users sharing the jump pod will not have their metadata overwrite the existing pod and service's metadata.